clouddrift/sphere.py

```diff
@@ -660,13 +664,13 @@ def cartesian_to_spherical(
     return lon, lat
 
 
+T = TypeVar("T", bound=float | np.ndarray)
+V = TypeVar("V", bound=float | np.ndarray)
+
+
 def cartesian_to_tangentplane(
-    u: float | np.ndarray,
-    v: float | np.ndarray,
-    w: float | np.ndarray,
-    longitude: float | np.ndarray,
-    latitude: float | np.ndarray,
-) -> tuple[float, float] | tuple[np.ndarray, np.ndarray]:
+    u: T, v: T, w: T, longitude: V, latitude: V
```

Comment:
but I'm confused then what is the puporse of the `ArrayTypes` from `clouddrift.typing` if we use `TypeVar` here.

Reply:
Here we define bounds to the type variable so while the type is "variable" we can define some bounds to constrain it a bit

Reply:
I am curious as to why we have both T and V here when they seem to have the same definition.